build.gradle

```diff
@@ -690,6 +690,7 @@ project(':streams') {
     testCompile project(':core')
     testCompile project(':core').sourceSets.test.output
     testCompile libs.junit
+    testCompile 'com.github.jbellis:jamm:0.3.1'
```

Comment:
Still exploratory as far as I know.  The only value in merging is if we want to merge in the benchmarking code for documentation purposes.